dotnet-roslyn/config/Audacia.CodeAnalysis/CHANGELOG.md

```diff
@@ -0,0 +1,22 @@
+﻿# Changelog
+
+## 1.8.0.0 - 2024-02-22
```

Comment:
I'm not 100% sure what the v1.9.0 version includes, I think it is to add Analyzers 1.9.0 so would add "DoNotUseNumberInIdentifierName" rule?